app/amity.py

```diff
@@ -1,49 +1,72 @@
+from fellow import Fellow
+from staff import Staff
+from office import Office
+from living_space import LivingSpace
+
+import random
```

Comment:
If you look at PEP8 https://www.python.org/dev/peps/pep-0008/#imports, you will see that STDLib imports should always come first.